classes/Settings.cs

```diff
@@ -77,7 +77,7 @@ public class Settings
 		public enum Platform { None, TIE, XvT, BoP, XWA, XWING }
 		public enum StartupMode { Normal, LastPlatform, LastMission }
 		[Flags]
-		public enum MapOpts { None, FGTags, Traces }
+		public enum MapOpts { None = 0, FGTags = 1, Traces = 2, TraceDistance = 4, TraceTime = 8, TraceHideFade = 16, TraceSelected = 32 }
```

Comment:
Adding the values is unnecessary, that's the purpose of "[Flags]" and it's handled in the background.

Reply:
No, [Flags] doesn't do that.  I assumed it did at first, but it wasn't working correctly and had to investigate.  The enum values would be {1, 2, 3...} which worked in your original code because it only uses 1 and 2 and that happens to be correct.  They do have to be explicit powers of two, in order to work correctly.  Apparently the [Flags] attribute only affects the output of ToString().

Reply:
Huh, while that seems dumb at first, I see that allows the ability to create hybrid flag enum values so you don't have to combine common combinations every time.